app/src/main/res/values-my/strings.xml

```diff
@@ -0,0 +1,887 @@
+    <string name="address_found">Ethereum လိပ်စာ</string>
+    <string name="is_address">Ethereum လိပ်စာကိုဖတ်ပြီးပါပြီ။ တိုကင်အဖြစ်ထည့်သွင်းရန်ကြိုးစားကြည့်ပါမည်လား။</string>
+    <string name="dialog_load_as_contract">တိုကင်ကိုဖော်ပြမည်</string>
+    <string name="blockchain">%1$s Blockchain</string>
+    <string name="currency_00">0.00 %s</string>
+    <string name="telegram">Telegram</string>
+    <string name="transfer_request">အပြောင်းအရွှေ့တောင်းဆိုမှု</string>
+    <string name="token_transfer_request">တိုကင်အပြောင်းအရွှေ့တောင်းဆိုမှု: %1$s</string>
+    <string name="generate_payment_request">ငွေပေးချေရန်တောင်းဆိုမှုကိုပြုလုပ်မည်</string>
+    <string name="enter_the_requested_amount">တောင်းမည့်ပမာဏကိုထည့်ပါ:</string>
+    <string name="feemaster_service_not_available">Feemaster ဝန်ဆောင်မှုကိုလတ်တလောတွင်မရနိုင်သေးပါ။ နောက်ထပ်ကြိုးစားကြည့်ပါ။ </string>
+    <string name="importing_tickets">တိုကင်များထည့်သွင်းနေသည်</string>
+    <string name="wallet_name">ဝေါလက်နာမည်</string>
+    <string name="wallet_name_template">Wallet %1$d</string>
+    <string name="no_token_found_title">တိုကင်ကိုရှာမတွေ့ပါ</string>
+    <string name="no_token_found">ယခုလိပ်စာနှင့်ပတ်သက်၍Ethereum Chain ပေါ်တွင်မည်သည့်တိုကင်မျှရှာမတွေ့ပါ။</string>
+    <string name="welcome_to_the_decentralized_web">ဗဟိုချုပ်ကိုင်မှုမရှိသောဝက်ဘ်စာမျက်နှာမှကြိုဆိုပါသည်</string>
+    <string name="action_remove">ဖယ်ရှားသည်</string>
+    <string name="title_remove_dapp">DAppကိုဖယ်ရှားသည်</string>
+    <string name="remove_from_my_dapps">%1$sကို ကျွန်ုပ် Dappမှဖယ်ရှားပါမည်လား။</string>
+    <string name="dialog_title_clear">မှတ်တမ်းများကိုရှင်းလင်းပါ</string>
+    <string name="action_clear">ရှင်းလင်းမည်</string>
+    <string name="dialog_message_clear">Browserပေါ်ရှိမှတ်တမ်းများကိုရှင်းလင်းရန်သေချာပါသလား။</string>
+    <string name="your_browser_history_appears_here">Browser၏မှတ်တမ်းများကိုဒီနေရာတွင်ဖော်ပြပါမည်။</string>
+    <string name="browser_history">Browser သုံးစွဲမှုမှတ်တမ်း</string>
+    <string name="you_have_no_dapps_displayed_yet_start_by_surfing_the_browser">You have no DApps displayed yet.\nStart by surfing the browser.</string>
+    <string name="my_dapps">My DApps</string>
+    <string name="add">ထည့်မည်</string>
+    <string name="edit">တည်းဖြတ်မည်</string>
+    <string name="history">မှတ်တမ်း</string>
+    <string name="discover_dapps">DAppsများကိုရှာဖွေမည်</string>
+    <string name="action_add">ထည့်မည်</string>
+    <string name="add_to_my_dapps">ကျွန်ုပ်၏DAppsသို့ထည့်မည်</string>
+    <string name="edit_dapp">DAppကိုတည်းဖြတ်မည်</string>
+    <string name="dapp_title">ခေါင်းစဥ်</string>
+    <string name="dapp_address">လိပ်စာ</string>
+    <string name="search_or_type_web_address">ဝက်ဘ်လိပ်စာကိုရိုက်ထည့်ရှာဖွေပါ</string>
+    <string name="network">ကွန်ရက်</string>
+    <string name="select_active_networks">အသက်ဝင်နေသောကွန်ရက်များကိုရွေးချယ်ပါ</string>
+    <string name="collectibles">စုဆောင်ထားသောအရများ</string>
```

Comment:
I would like to make translation correction "စုဆောင်းထားသောအရာများ" instead of "စုဆောင်ထားသောအရများ".
But in the wallet tab, Myanmar Translation is too long.